test/contestant_test.rb

```diff
@@ -0,0 +1,81 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/contestant'
+
+class ContestantTest < Minitest::Test
+  def test_it_exists
+    alexander = Contestant.new({
+      first_name: 'Alexander',
+      last_name: 'Aigiades',
+      age: 28,
+      state_of_residence: 'CO',
+      spending_money: 10
+    })
+    assert_instance_of Contestant, alexander
+  end
+
+  def test_it_has_attributes
+    alexander = Contestant.new({
+      first_name: 'Alexander',
+      last_name: 'Aigiades',
+      age: 28,
+      state_of_residence: 'CO',
+      spending_money: 10
+    })
+
+    assert_equal 28, alexander.age
+    assert_equal "CO", alexander.state_of_residence
+    assert_equal 10, alexander.spending_money
+  end
+
+  def test_contestant_has_full_name
+    alexander = Contestant.new({
+      first_name: 'Alexander',
+      last_name: 'Aigiades',
+      age: 28,
+      state_of_residence: 'CO',
+      spending_money: 10
+    })
+
+    assert_equal "Alexander Aigiades", alexander.full_name
+  end
+
+  def test_contestant_is_not_out_of_state
+    alexander = Contestant.new({
+      first_name: 'Alexander',
+      last_name: 'Aigiades',
+      age: 28,
+      state_of_residence: 'CO',
+      spending_money: 10
+    })
+
+    assert_equal false, alexander.out_of_state?
```

Comment:
If a method should return either false OR true, its best to write assertions for both scenarios :)